core/src/components/button/button.tsx

```diff
@@ -112,16 +115,18 @@ export class Button implements ComponentInterface, AnchorInterface, ButtonInterf
   /**
    * Set to `"round"` for a button with more rounded corners.
    */
-  @Prop({ reflect: true }) shape?: 'round';
+  @Prop({ reflect: true }) shape?: 'round' | 'rectangular';
 
   /**
    * Set to `"small"` for a button with less height and padding, to `"default"`
    * for a button with the default height and padding, or to `"large"` for a button
    * with more height and padding. By default the size is unset, unless the button
    * is inside of an item, where the size is `"small"` by default. Set the size to
    * `"default"` inside of an item to make it a standard size button.
+   * Option `"xsmall"` and `"xlarge"` are only available for Ionic theme.
```

Comment:
Suggestion:

> `"xsmall"` is only available for the `"ionic"` theme.
> `"xlarge"` is only available for the `"ionic"` theme. 

I don't believe we need to specify what happens on iOS or MD here. I suspect we will later fill in values for those themes 👍

Reply:
Indeed! This line was removed...